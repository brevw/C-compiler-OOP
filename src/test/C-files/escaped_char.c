#include "minic-stdlib.h"

int main() {
    int a = 1;
    char b = '\t';
    return a;
}
