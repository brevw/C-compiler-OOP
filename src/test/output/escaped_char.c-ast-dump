Program(FunDef(INT,main,Block(VarDecl(INT,a),VarDecl(CHAR,b),ExprStmt(Assign(VarExpr(a),IntLiteral(1))),ExprStmt(Assign(VarExpr(b),ChrLiteral(	))),Return(VarExpr(a)))))