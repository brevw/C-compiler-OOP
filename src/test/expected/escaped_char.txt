INCLUDE
STRING_LITERAL(minic-stdlib.h)
INT
IDENTIFIER(main)
LPAR
RPAR
LBRA
INT
IDENTIFIER(a)
SC
CHAR
IDENTIFIER(b)
SC
IDENTIFIER(a)
ASSIGN
INT_LITERAL(1)
SC
IDENTIFIER(b)
ASSIGN
CHAR_LITERAL(	)
SC
RETURN
IDENTIFIER(a)
SC
RBRA
Lexing: pass
